content/pt-br/docs/concepts/overview/working-with-objects/common-labels.md

```diff
@@ -54,17 +54,17 @@ metadata:
 
 ## Aplicações e Instâncias de Aplicações
 
-Um aplicativo pode ser instalado uma ou mais vezes em um cluster Kubernetes e,em alguns casos, com o mesmo *namespace*. Por exemplo, o WordPress pode ser instalado mais de uma vez, sendo que diferentes sites são diferentes instalaçãoes do Wordpress.
+Um aplicativo pode ser instalado uma ou mais vezes em um cluster Kubernetes e, em alguns casos, com o mesmo *namespace*. Por exemplo, o WordPress pode ser instalado mais de uma vez, sendo que diferentes sites são diferentes instalações do Wordpress.
 
-O nome da aplicação e o nome da instância são gravadas separadamente, Por exemplo, o Wordpress tem a `app.kubernetes.io/name` do `wordpress` enquanto tem o nome da instância representada por `app.kubernetes.io/instance` com o valor de `wordpress-abcxzy`. Isso permite que a aplicação e a instância da aplicação possa ser identificada. Toda instância de aplicação deve ter um nome único.
+O nome da aplicação e o nome da instância são gravados separadamente. Por exemplo, o Wordpress tem a `app.kubernetes.io/name` do `Wordpress` enquanto tem o nome da instância representada por `app.kubernetes.io/instance` com o valor de `wordpress-abcxzy`. Isso permite que a aplicação e a instância da aplicação possa ser identificada. Toda instância de aplicação deve ter um nome único.
```

Comment:
In this sentence, how about having the word _label_ appear just before the labels? Such as:
> ... o WordPress tem a _label_ `app.kubernetes.io/name`...

I think this would make the sentence more fluid for Portuguese readers. Same applies to `app.kubernetes.io/instance`:
> ... nome da instância representada pela _label_ `app.kubernetes.io/instance`...